modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/frontend/data/set/action/ReactFDSCreationMenu.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.sample.web.internal.frontend.data.set.action;
+
+import com.liferay.frontend.data.set.FDSEntryItemImportPolicy;
+import com.liferay.frontend.data.set.action.FDSCreationMenu;
+import com.liferay.frontend.data.set.sample.web.internal.constants.FDSSampleFDSNames;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.CreationMenu;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.CreationMenuBuilder;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemBuilder;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.language.Language;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Marko Cikos
+ */
+@Component(
+	property = "frontend.data.set.name=" + FDSSampleFDSNames.REACT,
+	service = FDSCreationMenu.class
+)
+public class ReactFDSCreationMenu implements FDSCreationMenu {
+
+	@Override
+	public CreationMenu getCreationMenu(
+			HttpServletRequest httpServletRequest,
+			HttpServletResponse httpServletResponse)
+		throws PortalException {
+
+		return CreationMenuBuilder.addPrimaryDropdownItem(
+			DropdownItemBuilder.setIcon(
+				"date"
+			).setLabel(
+				_language.get(httpServletRequest, "date")
+			).build()
+		).addPrimaryDropdownItem(
+			DropdownItemBuilder.setIcon(
+				"time"
+			).setLabel(
+				_language.get(httpServletRequest, "time")
+			).build()
+		).build();
+	}
+
+	@Override
+	public FDSEntryItemImportPolicy getFDSEntryItemImportPolicy() {
+		return FDSEntryItemImportPolicy.GROUP_PROXY;
+	}
```

Comment:
I added new sample to have three samples with different import policy each.

Reply:
Good! we'll grow these once all rendering/serialization logic is unified.